client/js/team.ts

```diff
@@ -20,7 +20,7 @@ joinTeamButton && joinTeamButton.addEventListener("click", e => {
 		await sweetAlert("Awesome!", "You successfully joined your new team.", "success");
 		window.location.assign("/team");
 	}).catch(async (err: Error) => {
-		await sweetAlert("Oh no!", "No such team", "error");
+		await sweetAlert("Oh no!", err.message, "error");
```

Comment:
:+1: This can stay as-is because it's referencing the message property of the Error object.